ETL/Python ETL/MatchHistoryETL.py

```diff
@@ -0,0 +1,266 @@
+                 "VCCH", "VCCD", "VCCA", "MaxCH", "MaxCD", "MaxCA", "AvgCH", "AvgCD", "AvgCA", "B365C>2.5", "B365C<2.5", "PC>2.5",
+                 "PC<2.5", "MaxC>2.5", "MaxC<2.5", "AvgC>2.5", "AvgC<2.5", "AHCh", "B365CAHH", "B365CAHA", "PCAHH", "PCAHA", "MaxCAHH",
+                 "MaxCAHA", "AvgCAHH", "AvgCAHA"]
+
+website_frame.drop(columns = unwanted_cols, inplace = True)
+
+
+# In[7]:
+
+
+# Rename columns
+website_frame.rename(columns = {"FTHG": "FullTimeHomeTeamGoals",
+                               "FTAG": "FullTimeAwayTeamGoals",
+                               "FTR": "FullTimeResult",
+                               "HTHG": "HalfTimeHomeTeamGoals",
+                               "HTAG": "HalfTimeAwayTeamGoals",
+                               "HTR": "HalfTimeResult",
+                               "HS": "HomeTeamShots",
+                               "AS": "AwayTeamShots",
+                               "HST": "HomeTeamShotsOnTarget",
+                               "AST": "AwayTeamShotsOnTarget",
+                               "HF": "HomeTeamFouls",
+                               "AF": "AwayTeamFouls",
+                               "HC": "HomeTeamCorners",
+                               "AC": "AwayTeamCorners",
+                               "HY": "HomeTeamYellowCards",
+                               "AY": "AwayTeamYellowCards",
+                               "HR": "HomeTeamRedCards",
+                               "AR": "AwayTeamRedCards",
+                               "B365H": "B365HomeTeam",
+                               "B365D": "B365Draw",
+                               "B365A": "B365AwayTeam",
+                               "MaxH": "MarketMaxHomeTeam",
+                               "MaxD": "MarketMaxDraw",
+                               "MaxA": "MarketMaxAwayTeam",
+                               "AvgH": "MarketAvgHomeTeam",
+                               "AvgD": "MarketAvgDraw",
+                               "AvgA": "MarketAvgAwayTeam",
+                               "B365>2.5": "B365Over2.5Goals",
+                               "B365<2.5": "B365Under2.5Goals",
+                               "Max>2.5": "MarketMaxOver2.5Goals",
+                               "Max<2.5": "MarketMaxUnder2.5Goals",
+                               "Avg>2.5": "MarketAvgOver2.5Goals",
+                               "Avg<2.5": "MarketAvgUnder2.5Goals"},
+                   inplace = True)
+
+
+# In[8]:
+
+
+# Add MatchID column
+
+website_frame.insert(0, "MatchID", constants.CURRENT_SEASON_TAG + "_" + website_frame["HomeTeam"] + "_" + website_frame["AwayTeam"])
+
+
+# In[9]:
+
+
+# Add season column
+
+website_frame.insert(1, "Season", constants.CURRENT_SEASON_TAG)
+
+
+# In[10]:
+
+
+# Stablish a connection to Database data source and fetch last game so we can know current matchweek
+
+try:
```

Comment:
can you create a config file for this

Reply:
Do you mean a yaml file for connecting to the database?